src/AlertBox/AlertBox.jsx

```diff
@@ -0,0 +1,70 @@
+import _ from "lodash";
+import classnames from "classnames";
+import React, {PureComponent, PropTypes} from "react";
+
+import "./AlertBox.less";
+import {CloseIcon, WarningIcon, SuccessIcon, ErrorIcon, InfoIcon} from "./Icons";
+
+/*
+ * AlertBox is a closable, highlighted box
+*/
+export default class AlertBox extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {isOpen: true};
+  }
+
+  closeBox() {
+    this.setState({isOpen: false});
+  }
+
+  render() {
+    const {children, className, type, title, isClosable} = this.props;
+    const {isOpen} = this.state;
+    const {Icons, cssClass} = AlertBox;
+    if (!isOpen) {
+      return null;
+    }
+    return (
+      <div className={classnames(`AlertBox--${type}`, cssClass.CONTAINER, className)}>
+        <div className={cssClass.HEADER}>
+          <span>{Icons[type]()}</span>
+          <strong className={cssClass.TITLE}> {title} </strong>
+          {isClosable &&
+            <button className={cssClass.CLOSE} onClick={() => this.closeBox()}><CloseIcon /></button>
+          }
+        </div>
+        {children}
+      </div>
+    );
+  }
+}
+
+AlertBox.Type = {
+  WARNING: "warning",
+  SUCCESS: "success",
+  ERROR: "error",
+  INFO: "info",
+};
+
+AlertBox.Icons = {
+  warning: WarningIcon,
+  success: SuccessIcon,
+  error: ErrorIcon,
+  info: InfoIcon,
+};
+
+AlertBox.cssClass = {
```

Comment:
to keep with the plural/non-member naming scheme above, `const CLASSES` or `CLASS_NAMES`; I wouldn't append `css` since classnames are as much a part of the HTML spec as they are in CSS.